x-pack/libs/es-opensaml-security-api/build.gradle

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+apply plugin: 'elasticsearch.build'
+apply plugin: 'com.github.johnrengelman.shadow'
+
+dependencies {
+  implementation "org.opensaml:opensaml-security-api:${versions.opensaml}"
+  compileOnly "org.opensaml:opensaml-core:${versions.opensaml}"
+}
+
+['jarHell', 'thirdPartyAudit', 'forbiddenApisMain', 'splitPackagesAudit', 'checkstyleMain', 'licenseHeaders', 'spotlessJavaCheck'].each {
```

Comment:
@mark-vieira I see that `forbiddenApisMain` fails with:

```
> Task :x-pack:libs:es-opensaml-security-api:forbiddenApisMain FAILED

> Task :libs:elasticsearch-core:compileJava
Note: Some input files use or override a deprecated API that is marked for removal.
Note: Recompile with -Xlint:removal for details.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':x-pack:libs:es-opensaml-security-api:forbiddenApisMain'.
> Parsing signatures failed: Class 'org.apache.lucene.util.IOUtils' not found on classpath while parsing signature: org.apache.lucene.util.IOUtils
```

Is it a real problem? I suspect it's just the forbidden task itself missing a dependency classs?

Reply:
_I've only reenable `checkstyle` and `spotless` for now_

Reply:
Yeah, this looks to be because some of our forbidden api rules reference classes that aren't on the classpath here. We can disable this task here for now.